ui/settings.py

```diff
@@ -343,3 +343,8 @@
         'promo-video_web.mp4'
     )
 )
+
+# directory CMS
+CMS_URL = os.environ['CMS_URL']
+CMS_SIGNATURE_SECRET = os.environ['CMS_SIGNATURE_SECRET']
+FEATURE_CMS_ENABLED = True
```

Comment:
this should be feature flagged and off by default.

Reply:
whoops!